src/OrchardCore/OrchardCore.DisplayManagement.Liquid/TagHelpers/LiquidTagHelperActivator.cs

```diff
@@ -58,11 +57,23 @@ public LiquidTagHelperActivator(Type type)
                     {
                         allNames.Add(htmlAttribute.Name.Substring(4).ToPascalCaseDash());
                     }
+                    var dictonaryPrefix = htmlAttribute.DictionaryAttributePrefix;
+                    if (dictonaryPrefix != null)
+                    {
+                        allNames.Add(dictonaryPrefix.ToPascalCaseDash());
+
+                        if (dictonaryPrefix.StartsWith("asp-", StringComparison.Ordinal))
```

Comment:
We could use a `private const string AspPrefix = "asp-";` and then `AspPrefix.Length`.
And use it also above when we do `htmlAttribute.Name.StartsWith("asp-", ...)`

Reply:
Added private const and replaced at related location